config/locales/en.yml

```diff
@@ -30,7 +30,7 @@ en:
         search:
           attributes:
             keywords:
-              blank: Keywords can't be blank
+              blank: Please enter Keywords and/or a valid date range
```

Comment:
Given what Ian was saying about not making the service too easy to use, is there an argument that we shouldn't be highlighting that you can search with just a date range?

Reply:
Agreed, I've reverted this text change.